user-guide/Advanced_Modules/DOM/DOM_objects/DOM_FieldDescriptors/DOM_AutoIncrementFieldDescriptor.md

```diff
@@ -0,0 +1,40 @@
+---
+uid: DOM_AutoIncrementFieldDescriptor
+---
+
+# AutoIncrementFieldDescriptor
+
+- **FieldValue type**: string
+- **Multiple values supported**: :heavy_multiplication_x:
+
+| Type of Descriptor | FieldType | FieldValue type |
+|--------------------|-----------|-----------------|
+| AutoIncrement | string | string |
+
+Defines a field that will automatically get an incremented value assigned. When a DomInstance does not have a value for this field yet, it will get assigned the next time the instance is updated.
```

Comment:
```suggestion
Defines a field that will automatically get an incremented value assigned. When a `DomInstance` does not have a value for this field yet, it will get assigned the next time the instance is updated.
```